cs_api_io/cs_api_io.ml

```diff
@@ -77,3 +77,25 @@ let get_response {Response.code; body} =
   else
     let message = Printf.sprintf "HTTP response: %d\n%s" code body in
     Lwt_result.fail message
+
+let get_error_message (errors : Yojson.Safe.t list) =
+  match CCList.hd errors with
+  | `Assoc json -> (
+    CCList.find_opt (fun (m, _) -> m = "message") json |> function
```

Comment:
Perhaps it would be interesting to use `Yojson.Safe.Util.member` here?

Reply:
I did not know about this function. Should probably have known something like that existed already.

I also replaced the other use of `CCList.find_opt`. Do you like it better this way?